application/backup.sh

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env sh
+# Error Tracking
+if [ ! -z "${ SENTRY_DSN }"]; then
```

Comment:
Spaces are critical and make a difference in how it is interpreted.
```suggestion
if [ ! -z "${SENTRY_DSN}" ]; then
```